### DISCUSSION TITLE
art/visual graphic ideas

### DISCUSSION THREAD

**Discussion by kodriq** [jeffaroundtheworld/2024gamecollab#9] [General]
put image examples of art or sprites or whatever from games that u would want our graphics to be inspired by
(specifically for the enemies)

**Comment by kodriq** [comment]
also should the enemies be 2D or 3D
cause I've seen 2D graphics in a 3D world and it normally looks cool
3D would still be fine but personally idk how to make 3D models

**Reply by jeffaroundtheworld** [reply]
Check out [Mixamo ](https://www.mixamo.com/)for some help with generating 3D character models.  You can select from a vast array of models and then assign a vast array of movements.  These can be brought in to Godot via Blender - would take about 30 mins to learn how to do.  You can get some pretty stunning results.  Here I put one into the 3D game we are doing with the 11s.

[Recording-20240206_153443.webm](https://github.com/jeffaroundtheworld/2024gamecollab/assets/48102368/fc68490e-9887-4229-aed5-28ec1b1fb5a1)

**Reply by Rhyswatsonn** [reply]
i guess it depends what kind of visuals we want. we could go for a more retro like pixelized but not at the same time, or we could go for a cartoon look or realistic

**Reply by zorienm** [reply]
I think we could probably do 3D

**Reply by kodriq** [reply]
I think having our own unique models would be cool andif we wanna do 3D i can look into how difficult it would be to learn to make them
I lowkey doubt wed actually be able to learn how to make one this quick so maybe we should have a poll over having premade 3D models or original 2D models

**Reply by Rhyswatsonn** [reply]
i was thinking if we put 2d graphics and wrap them on 3d models it could be cool because it would kinda warp the desings

**Reply by kodriq** [reply]
that would be cool, do u have any examples of that?

**Reply by Rhyswatsonn** [reply]
![image](https://github.com/jeffaroundtheworld/2024gamecollab/assets/159056791/09e79452-0cc2-4e59-b9a4-0a878f87af91)
![image](https://github.com/jeffaroundtheworld/2024gamecollab/assets/159056791/fb87ab28-415a-4475-be74-4f7a55adc972)
![image](https://github.com/jeffaroundtheworld/2024gamecollab/assets/159056791/15aff95c-ca79-4710-ad47-f09e9d7d680f)
this isnt exactly what i was talking about but if we took texures like theese and wraped them on characters. Zach aslo said we could do 2d enimes but we can code them to always face you, would end up looking good. maybe we coiuld vote on it

**Reply by kodriq** [reply]
i really like that pixelated look tbh that is cool
and alr ill make a poll

**Comment by jeffaroundtheworld** [comment]
IF you end up going with the 2D Doom-style graphics, someone could play around with Adobe Character Animator and create some horror style puppets that could be used in cut scenes to forward your narrative.  This could allow you to quickly add backstory, lore, and ability to further the plot without a tonne of time doing the animation.

Here is a video on how it works.  I have bought a webcam for the class if needed.

[https://www.youtube.com/watch?v=7GKCw4f1UrI&t=0s](https://www.youtube.com/watch?v=7GKCw4f1UrI&t=0s)
